docs/pages/setup/reference/networking.mdx

```diff
@@ -18,22 +22,51 @@ following use cases:
 
 - You have multiple identical services, like proxies, behind a load balancer.
 - You want Teleport to issue an SSH certificate for the service with the additional
-  principals, e.g.host names.
+  principals, e.g., host names.
+</TabItem>
+<TabItem scope={["cloud"]} label="Teleport Cloud">
+
+Teleport Nodes have an optional `public_addr` property that you can modify in
+each service's configuration file. The public address can take an IP or a DNS
+name. It can also be a list of values:
+
+```yaml
+public_addr: ["proxy-one.example.com", "proxy-two.example.com"]
+```
+
+Specifying a public address for a Teleport Node may be useful in the
+following use cases:
+
+- You have multiple identical services, like proxies, behind a load balancer.
+- You want Teleport to issue an SSH certificate for the service with the additional
+  principals, e.g., host names.
+</TabItem>
+</Tabs>
+
 
 ## HTTP CONNECT proxies
 
 Some networks funnel all connections through a proxy server where they can be
-audited and access control rules are applied. For these scenarios, Teleport supports HTTP CONNECT tunneling.
+audited and access control rules can be applied. For these scenarios, Teleport
+supports HTTP CONNECT tunneling.
+
+To use HTTP CONNECT tunneling, simply set either the `HTTPS_PROXY` or
+`HTTP_PROXY` variables of the `teleport` daemon's shell environment. When the
```

Comment:
```suggestion
`HTTP_PROXY` variables of the `teleport` daemon's environment. When the
```

Remove `shell` as there is no requirement to run the Teleport daemon via a shell.